CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Changes since last non-beta release.
 
 *Please add entries here for your pull requests that are not yet released.*
 - Added `./bin/dev` and `./bin/dev-static` executables to ease and standardize running the dev server. [PR 1491](https://github.com/shakacode/react_on_rails/pull/1491) by [ahangarha](https://github.com/ahangarha)
+- Fixed pack not found warning while using `react_component` and `react_component_hash` helpers, even when corresponding chunks are present in `manifest` file [PR 1511](https://github.com/shakacode/react_on_rails/pull/1511) by [pulkitkkr](https://github.com/pulkitkkr)
```

Comment:
what does the manifest have to do with the changes?

Reply:
Updated it to: 
```
Fixed pack not found warning while using `react_component` and `react_component_hash` helpers, even when corresponding chunks are present.
```